src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -6,7 +6,10 @@
 using System.Runtime.CompilerServices;
 using System.Numerics;
 
+#if !netstandard
```

Comment:
These netstandard ifdefs are no longer needed.

Reply:
Would I need to add a ref to `System.Runtime.Intrinsics.X86` in corefx instead?

Reply:
These files are not compiled in corefx anymore. We stopped building the down-level System.Memory.

Reply:
Makes it easy then, removed